src/Propel/Runtime/Connection/PdoConnection.php

```diff
@@ -137,13 +137,11 @@ public function query($statement)
     /**
      * @inheritDoc
      *
-     * @return \Propel\Runtime\DataFetcher\DataFetcherInterface
+     * @return int
```

Comment:
could you also adjust an interface if this change is really needed?

Reply:
Right, yes, good point. Adjusted the interface declaration in ConnectionInterface